runtime/src/anchor/tests.rs

```diff
@@ -51,6 +51,7 @@ impl pallet_timestamp::Trait for Test {
 
 impl fees::Trait for Test {
     type Event = ();
+    type FeeChangeOrigin = frame_system::EnsureRoot<u64>;
```

Comment:
I am confused as to why Root would work here - or is the council automatically treated as root?

Reply:
yeah... brainfart.... this is the test file. all good.